ci-scripts/genrelnotes.py

```diff
@@ -40,12 +40,14 @@ def extract_release_notes(git_output, tag):
         sys.exit(1)
 
     titles = {
-        'feat': '## New features and enhancements',
+        'feat': '## New features',
+        'enhancement': '## Enhancements',
+        'doc': '## Enhancements',
         'bugfix': '## Bug fixes',
         'testlib': '## Test library'
     }
     sections = collections.OrderedDict()
-    for tag in ['feat', 'bugfix', 'testlib', 'ci', 'doc']:
+    for tag in ['feat', 'enhancement', 'doc', 'bugfix', 'testlib', 'ci']:
```

Comment:
```suggestion
    for tag in titles.keys():
```

Reply:
The reason I did that was that the `ci` tag didn't have a dedicated section, removing it from here would make it not to appear in the release notes. I changed this loop completely to make the intent clearer.